effective-altruism-wiki.csl

```diff
@@ -0,0 +1,1167 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="4" et-al-use-first="1" initialize="true" page-range-format="expanded" demote-non-dropping-particle="never">
```

Comment:
Please run your style through https://formatter.citationstyles.org/ to fix the indenting